api/src/routes/public/deprecated-endpoints.ts

```diff
@@ -8,6 +8,8 @@ export const endpoints: Endpoints = [
   ['/refetch-user-completed-challenges', 'POST'],
   ['/certificate/verify-can-claim-cert', 'GET'],
   ['/api/github', 'GET'],
+  ['/api/users/exists', 'GET'],
+  ['/api/users/get-public-profile', 'GET'],
```

Comment:
I am still fuzzy on the "why" for removing the `/api` stuff. But, I thought having it respond as a deprecated endpoint would cause the same issue as having it exist?

Reply:
I wasn't sure about that - I just replicated what we did last time we "removed" an endpoint.

@raisedadead do you just want the `/api` gone?